aurpublish.in

```diff
@@ -147,6 +147,18 @@ fi
 if [[ "$url_set" != "true" ]]; then
     # use git configured remoteUrl, if set
     REMOTE_URL=$(git config --default "${REMOTE_URL}" --get aurpublish.remoteUrl | sed 's:/*$::')
+
+    # warn the user if they have a non-default ssh alias for 'aur'
+    auralias=$(ssh -G aur | awk '$1 == "hostname" { print $2 }')
+    if [[ "$auralias" != 'aur' && "$auralias" != 'aur.archlinux.org' ]]; then
+        echo "Warning: You have a non-default alias for 'aur' in your ssh configuration. (${auralias})"
+        echo "Previous versions of aurpublish used the 'aur' ssh hostname, but you should now use the --url option."
+        echo ""
+        echo "This operation will use the git server at: ${REMOTE_URL}"
+        echo ""
+        read -p "Do you wish to continue? [y/n] " userwarned
+        [[ $userwarned == [yY] ]] || exit 1
```

Comment:
Avoid using `==` in the `[[` shell keyword. It is a bash-specific alias for `=`, which is nominally okay since this is a bash script, but it also has zero utility and corrupts muscle memory if you write POSIX sh scripts as well. This alias should never have existed IMO.

Reply:
Fixed!

Also, I learned something new. I wasn't aware of the double-equal alias. Thank you for the education. I found that this mistake had also made it into a couple of my personal scripts due to muscle memory. I'd prefer it if bash didn't try to work around my mistake in this situation.

Reply:
:) My pleasure.